src/data/engineering/Navigation.tsx

```diff
@@ -1,6 +1,12 @@
 import { SiGithub as Github } from "@icons-pack/react-simple-icons";
+import React from "react";
 
-export const TABS = [
+type tab = {
+  label: string | React.ReactElement;
```

Comment:
should only be a string

Reply:
What about the icon label in one of the objects?